apps/cloudinary/src/index.js

```diff
@@ -153,6 +153,14 @@ setup({
       "type": "Number",
       "required": false,
       "default": 10
+    },
+    {
+      "id": "startFolder",
+      "name": "Starting folder",
+      "description": "Path to starting folder for the media library dialog",
```

Comment:
We should specify if the `path` needs to start with a `/`.

Reply:
The [docs](https://cloudinary.com/documentation/media_library_widget#3_set_the_configuration_options) don't say and it works without.